buildspec/linuxTests.yml

```diff
@@ -48,7 +48,8 @@ phases:
             - VCS_COMMIT_ID="${CODEBUILD_RESOLVED_SOURCE_VERSION}"
             - CI_BUILD_URL=$(echo $CODEBUILD_BUILD_URL | sed 's/#/%23/g') # Encode `#` in the URL because otherwise the url is clipped in the Codecov.io site
             - CI_BUILD_ID="${CODEBUILD_BUILD_ID}"
-            - test -n "${CODECOV_TOKEN}" && ./codecov --token=${CODECOV_TOKEN} --branch=${CODEBUILD_RESOLVED_SOURCE_VERSION} --repository=${CODEBUILD_SOURCE_REPO_URL} --file=./coverage/lcov.info
+            - test -n "${CODECOV_TOKEN}" && ./codecov --token=${CODECOV_TOKEN} --branch=${CODEBUILD_RESOLVED_SOURCE_VERSION} --repository=${CODEBUILD_SOURCE_REPO_URL} --file=./coverage/amazonq/lcov.info
```

Comment:
`--file` accepts multiple files according to https://docs.codecov.com/docs/cli-options

although that might mean either:

    --file=./coverage/amazonq/lcov.info --file=./coverage/core/lcov.info --file=./coverage/toolkit/lcov.info

or:

    --file ./coverage/*/lcov.info

I'm guessing it's the long one :( 

(Note: `=` is usually optional for most CLI args parsers, that's why the short one is at least syntactically valid)